### ISSUE TITLE
[Dungeon] Marudon - Constrictor Vine - Abilities

### ISSUE THREAD

**Issue by whammyresilience** [KurthosWoW/TrinityCore#98]
**Current behaviour:**

[Constrictor Vine](https://classicdb.ch/?npc=12220#abilities) is missing the following abilities: [Thrash](https://classicdb.ch/?spell=3391)

**Comment by Nyeriah** [comment]
https://github.com/KurthosWoW/TrinityCore/commit/7bf096290e0891b821232e8a16d599b353870b2e

**Comment by Glonz** [comment]
3391 is the incorrect ability. Whenever Constrictor Vine casts Thrash, it now hits itself with extra attacks.

Consider using **8876** instead. Just make it cast it on itself once when it is pulled and it will have a 15% chance to trigger 3391 and do attacks correctly.

**Comment by Nyeriah** [comment]
No, trash mechanic need to be fixed and hit the actual victim with the extra attacks, it's a global issue

**Comment by Glonz** [comment]
That might be true, however, I'd argue that extra attacks through Thrash should not occur on a fixed time, but rather randomly, as I've witnessed it in the Kurinnaxx encounter in AQ20. Surprisingly enough, if the mob casts 8876 instead of 3391, the aura actually works and the mob hits the target. (Combat log also shows that the spell will be 3391 if extra hits through Thrash occur, so that might add to the confusion.)

**Comment by whammyresilience** [comment]
Confirming this is still not working as intended: https://imgur.com/a/HLUj3gw
